Author: Luca Boccassi <bluca@debian.org>
Forwarded: https://github.com/systemd/systemd/pull/28303
Description: switch fuzz-manager-serialize to MANAGER_TEST_RUN_MINIMAL
 When there is no access to cgroups MANAGER_TEST_RUN_BASIC will fail
 to set up and assert. This happens on a build system like Debian's.
 Switch to _MINIMAL which skips cgroup and other machine-wide setups.
diff --git a/src/core/fuzz-manager-serialize.c b/src/core/fuzz-manager-serialize.c
index 04560341da..2f0c0c3029 100644
--- a/src/core/fuzz-manager-serialize.c
+++ b/src/core/fuzz-manager-serialize.c
@@ -21,7 +21,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
                 log_set_target(LOG_TARGET_NULL);
         }
 
-        assert_se(manager_new(RUNTIME_SCOPE_SYSTEM, MANAGER_TEST_RUN_BASIC, &m) >= 0);
+        assert_se(manager_new(RUNTIME_SCOPE_SYSTEM, MANAGER_TEST_RUN_MINIMAL, &m) >= 0);
         /* Set log overrides as well to make it harder for a serialization file
          * to switch log levels/targets during fuzzing */
         manager_override_log_level(m, log_get_max_level());
